index.html

```diff
@@ -1,140 +1,170 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<title>Git for Windows</title>
-		<meta name="description" content="We bring the awesome Git VCS to Windows">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
-		<link rel="stylesheet" href="css/pack.css">
```

Comment:
`pack.css` is built by [the `Gruntfile.js`](https://github.com/git-for-windows/git-for-windows.github.io/blob/main/Gruntfile.js).

If we do not use that any longer, we might just as well do away with `style.css` completely (rather than [changing it](https://github.com/git-for-windows/git-for-windows.github.io/pull/28/files#diff-1fc556f95754ee7e33d91044125c44bb9f750c99be4406756ffb27413adfcaf5) to no effect).

We would then also need to drop `grunt` and friends from [the `package.json` file](https://github.com/git-for-windows/git-for-windows.github.io/blob/main/package.json).

But then, how can we ensure a common look between `index.html`, `governance-model.html` and `requirements.html` (the latter two are generated from Markdown using the [`markdown-template.html`](https://github.com/git-for-windows/git-for-windows.github.io/blob/main/markdown-template.html) file).

In a similar vein, we render the Release Notes of Git for Windows using [this script](https://github.com/git-for-windows/build-extra/blob/HEAD/render-release-notes.sh) and [this CSS](https://github.com/git-for-windows/build-extra/blob/06cbe78b556354e4ef4822c9fa23211a449448d8/ReleaseNotes.css), which is a copy-edited version of the `css/style.css` file in this here repository. If we change the look of the home page, we will have to align the look of the release notes accordingly.